pkg/edgeview/src/tcp.go

```diff
@@ -72,7 +72,9 @@ type tcpConnRWMap struct {
 }
 
 const (
-	tcpMaxMappingNUM int = 5
+	tcpMaxMappingNUM       int           = 5
+	tcpIdleTimeoutDuration time.Duration = 1800
+	tcpCheckTimeout        time.Duration = 300
```

Comment:
time.Duration is nanoseconds I think. Did you mean to multiply by time.Second or something else?

Reply:
If you are using as seconds, then have the type be an int and not time.Duration because it is very confusing.

Reply:
yes, all the usage for the tcpIdleTimeoutDuration is to (* time.Second), since i can not define this as 'int'

Reply:
actually, let me change this to just int, since it's not used in the timer, only in comparison